### PR TITLE
Update of cisco product list part 1

### PR DESCRIPTION
**Pull request by anssi-cvp**
Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - [x] PR Title: Please use "Add <vendor/product name>" (instead of "Update README.md")
  - [x] Status: please select a value from the status table at the top
  - [x] Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - [x] Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - [x] Please mind the sorting

### CODE REVIEW

File: software/software_list_b.md [hunk 1]
```diff
@@ -545,59 +545,64 @@
...
+| Broadcom      | Layer7 API Developer Portal SaaS | 5.0.3 | Not vuln | Workaround | Workaround | | |[source](https://knowledge.broadcom.com/external/article?articleId=230205) |
+| Broadcom      | Layer7 API Gateway | 9.4, 10.0, 10.1 | Not vuln | Workaround | Workaround | | |[source](https://knowledge.broadcom.com/external/article?articleId=230205) |
 |Broadcom|Layer7 Live API Creator|5.4, 5.1-5.3 (EOS)|Not vuln|Workaround| | | |[source](https://knowledge.broadcom.com/external/article?articleId=230205)|
 |Broadcom|Layer7 Mobile API Gateway| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://support.broadcom.com/security-advisory/content/security-advisories/Symantec-Security-Advisory-for-Log4j-2-CVE-2021-44228-Vulnerability/SYMSA19793)|
-|Broadcom|LiveUpdate Administrator (LUA)|All supported versions|Not vuln|Workaround| | | |[source](https://support.broadcom.com/security-advisory/content/security-advisories/Symantec-Security-Advisory-for-Log4j-2-CVE-2021-44228-Vulnerability/SYMSA19793)|
+|Broadcom|LiveUpdate Administrator (LUA)|2.3.10| |Fix|Fix| | |[source](hthttps://support.broadcom.com/security-advisory/content/0/0/SYMSA19793)|
```
Comment (certeu) [inline software/software_list_b.md:571]: Issue in the link (https://support.broadcom.com/security-advisory/content/0/0/SYMSA19793)

File: software/software_list_c.md [hunk 2]
```diff
@@ -159,45 +165,70 @@
...
 |Cisco|Aironet 3800 Series Access Points| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
-|Cisco|AMP Virtual Private Cloud Appliance| | |Vulnerable| | |Fixes expecteded 10-Jan-2022|[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+| Cisco | Aironet Access Points | All versions | | Not vuln | Not vuln | Not vuln | |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+|Cisco|AireOS Wireless LAN Controllers | All versions | | Not vuln | Not vuln | Not vuln | |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+| Cisco | Cisco AMP Virtual Private Cloud Appliance | All versions | | Not vuln | Not vuln | Not vuln |Fixes expecteded 10-Jan-2022|[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
```
Comment (certeu) [inline software/software_list_c.md:170]: `Cisco` could be removed in the product column. Moreover, the comment ("Fixes expecteded 10-Jan-2022") could be removed as this devine is not vulnerable.

File: software/software_list_c.md [hunk 3]
```diff
@@ -159,45 +165,70 @@
...
 |Cisco|AnyConnect Secure Mobility Client|All|Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
-|Cisco|AppDynamics|<21.12.0|Not vuln|Fix| | | |[source](https://docs.appdynamics.com/display/PAA/Security+Advisory%3A+Apache+Log4j+Vulnerability)|
+| Cisco | AppDynamics | Multiple | Not vuln | Fix | Fix | Not vuln |See advisory for complete of of fixed versions|[source](https://docs.appdynamics.com/display/PAA/Security+Advisory%3A+Apache+Log4j+Vulnerability) |
```
Comment (certeu) [inline software/software_list_c.md:172]: Multiple `of` in the comment.

File: software/software_list_c.md [hunk 4]
```diff
@@ -159,45 +165,70 @@
...
+| Cisco | Business 350 Series Managed Switches | All versions | | Not vuln | Not vuln | Not vuln | |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd) |
+| Cisco | Cisco Business Dashboard | All versions | | Not vuln | Not vuln | Not vuln | |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd) |
+| Cisco | Business Process Automation | 3.0.000.115, 3.1.000.044, 3.2.000.009 | | Fix | Fix | Not vuln | |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd) |
+| Cisco | Business Wireless  | All versions | | Not vuln | Not vuln | Not vuln | |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd) |
+| Cisco | Cisco Call Studio | < 11.6, < 12.0, < 12.5, < 12.6 | | Vulnerable | Vulnerable | Not vuln | Fix will should be available from 23 Dec 2021 |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd) |
```
Comment (certeu) [inline software/software_list_c.md:188]: `Cisco` in the product Column could be removed. Also, we could maybe add the fixed version as the patch should be available 

11.6(2) (23 Dec 2021)
12.0(1) (23 Dec 2021)
12.5(1) (23 Dec 2021)
12.6(1) (23 Dec 2021)

File: software/software_list_c.md [hunk 5]
```diff
@@ -159,45 +165,70 @@
...
 |Cisco|Catalyst 9800 Series Wireless Controllers| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
 |Cisco|Wireless LAN Controller| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+| Cisco | CloudCenter | 4.10.0.16 | | Fix | Fix | Not vuln | Fixes should be available from 23 Dec 2021 |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd) |
```
Comment (certeu) [inline software/software_list_c.md:191]: The comment might be removed

File: software/software_list_c.md [hunk 6]
```diff
@@ -159,45 +165,70 @@
...
 |Cisco|CloudCenter Action Orchestrator| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
-|Cisco|CloudCenter Suite Admin| | |Vulnerable| | | |[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
-|Cisco|CloudCenter Workload Manager| | |Vulnerable| | |Fixes expecteded 23-Dec-2021|[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+| Cisco | CloudCenter Cost Optimizer | 5.5.2 |  | Fix | Fix | | Fix should be available from 23 Dec 2021 |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+| Cisco | CloudCenter Suite Admin | 5.3.1 |  | Fix | Fix | | Fix should be available from 23 Dec 2021 |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+|Cisco|CloudCenter Workload Manager| 5.5.2 | |Fix| | |Fixes expecteded 23-Dec-2021|[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
```
Comment (certeu) [inline software/software_list_c.md:195]: The comments for the 3 previous lines could be removed

File: software/software_list_c.md [hunk 7]
```diff
@@ -159,45 +165,70 @@
...
+| Cisco |Crosswork Zero Touch Provisioning (ZTP) | 2.0.1, 3.0.1 |  | Fix | Fix | Not vuln ||[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+| Cisco | CX Cloud | All versions | | Fix | Fix | Not vuln ||[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+| Cisco | CX Cloud Agent Software| 1.12.2 | | Fix | Fix | Not vuln ||[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+| Cisco | Cyber Vision Sensor Management Extension | 4.0.3 |  | Fix | Fix | Not vuln ||[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+| Cisco | Data Center Network Manager (DCNM) | 12.0(2d), 11.5(3), 11.5(2), 11.5(1), 11.4(1), 11.3(1) |  | Vulnerable | Vulnerable | Not vuln | Fix should be available from 23 Dec 2021 |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
```
Comment (certeu) [inline software/software_list_c.md:221]: Comment might be removed

File: software/software_list_c.md [hunk 8]
```diff
@@ -159,45 +165,70 @@
...
 |Cisco|Defense Orchestrator| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
 |Cisco|DNA Assurance| | |Investigation| | | |[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
-|Cisco|DNA Center| | |Vulnerable| | | |[source](https://tools.cisco.com/security/left/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
+| Cisco | DNA Center | 2.2.2.8, 2.1.2.8, 2.2.3.4 | | Fix | Fix | Not vuln | Fix should be available from 23 Dec 2021 for 2.2.2.8, 25 Jan 2022 for 2.1.2.8, 29 Dec 2021 for 2.2.3.4 |[source](https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-apache-log4j-qRuKNEbd)|
```
Comment (certeu) [inline software/software_list_c.md:224]: Comments might be removed.

**Review by certeu — CHANGES_REQUESTED** [review]

**Comment by certeu** [comment]
Hello @anssi-cvp, thank you so much for your contribution! Could you please have a look at our comments and do the modifications before we can merge?

**Comment by anssi-cvp** [comment]
Should be OK now :)

**Review by certeu — APPROVED** [review]

**Comment by certeu** [comment]
@anssi-cvp Thanks a lot for your correction and your contribution!

**Pull request merged**